improver/utilities/cube_manipulation.py

```diff
@@ -63,6 +63,11 @@ def collapsed(cube, *args, **kwargs):
     return new_cube
 
 
+def get_cube_dimensions(cube):
+    """Returns an ordered list of dimension coordinate names on the cube"""
```

Comment:
This docstring should probably have an `Args` and `Returns` section.

Reply:
It's a one-liner and with the new name and current docstring, it's not at all unclear what this is doing.  So I don't see any value to adding more words here.

Reply:
Adding formatted Args and Returns sections allows Sphinx to do clever stuff in linking the documentation with the specifications of a cube and a list. I can't see a good reason to bypass this.